src/api/app/controllers/statistics/maintenance_statistics_controller.rb

```diff
@@ -3,7 +3,7 @@ class MaintenanceStatisticsController < ApplicationController
     skip_before_action :extract_user
 
     def index
-      @project = Project.find_by(name: params[:project])
+      @project = Project.find_by!(name: params[:project])
```

Comment:
actually this is a good point to make, it reminds me that @ChrisBr and I discussed that this endpoint is only showing publicly available information so doesn't need to have an authentication. @hennevogel @adrianschroeter what do you guys think about removing the authorisation checks from this endpoint?